phpunit.integration.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.3/phpunit.xsd" backupGlobals="false" bootstrap="tests/bootstrap-integration.php" colors="false" processIsolation="false" stopOnError="false" stopOnFailure="false" stopOnIncomplete="false" stopOnSkipped="false" stopOnRisky="false" displayDetailsOnTestsThatTriggerDeprecations="true" displayDetailsOnTestsThatTriggerErrors="true" displayDetailsOnTestsThatTriggerWarnings="true" displayDetailsOnTestsThatTriggerNotices="true" testdox="false"> 
```

Comment:
just checking on why have different phpunit xml here?

Reply:
I wrote a separate phpunit file that doesn't run the globals.php bootstrap in the phpunit.  ApiApplication loads globals.php and since it can only be loaded using 'require_once' it messes up the SiteSetupListener class.  That's why you currently see SiteSetupListenerTest's functions commented out because ApiApplicationTest functions as the smokescreen test for the application.

I wanted to change up globals.php so it can just be a regular include (or move the entire functionality into a class) but I ended up not doing that in this round of changes.